### PR TITLE
23: Flower Footer Bug + Small Refactor

### PR DESCRIPTION
**Pull request by madeleine-patience**
https://github.com/madeleine-patience/Considerate-Cat-Version-2-With-Next/issues/23
# Pull Request Title

**🐾Description of Purrrrr Request🐾**

We want to display the flower image in all screen sizes. I have also simplified the css.

I have also added another mobile story to show this state as well as the ability to add mobile stories in the future.

* Also just quickly updated the github template to make it easier to do before afters.

### Type of Change

- [ ] Mew Feature added
- [ ] Bug fixed
- [🐱] Re-fur-ctor / Cleanup

### Description of Change

- Adjustments to CSS
- Small CSS refactor
- Added new story
- Added ability to add mobile viewports in future stories

### Screenshots or Demo (optional)
| Before | After |
|--------|-------|
| Before Image | After Image |

### How did you purrrsonally test this

Locally on storybook 

### Checklist

- [🐱] Tests added or updated
- [🐱] Documentation updated
- [🐱] Follows coding standards
- [🐱] Ready fur review

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -17,11 +17,13 @@
 - Please provide before and after or a screenshot of a new change you have made.
   Delete this section if not necessary to show screenshots.
 
-| Before | After |
-| ------ | ----- |
+| Before                                | After                              |
+| ------------------------------------- | ---------------------------------- |
+| ![Before Image]("Before Image Here" ) | ![After Image]("After Image Here") |
```
Comment (arthvadrr) [inline .github/PULL_REQUEST_TEMPLATE.md:22]: **praise:** Good idea!

File: components/flowerFooter/FlowerFooter.tsx [hunk 2]
```diff
@@ -77,15 +63,16 @@ const FlowerFooter = ({ sx }: flowerFooterProps) => {
...
                 display: 'flex',
+                flex: 1,
                 alignItems: 'center',
                 flexDirection: 'column',
-                textAlign: 'center'
+                textAlign: 'center',
+                maxWidth: 600,
+                p: 10,
+                gap: 4,
+                background: '#9C6E4F',
```
Comment (arthvadrr) [inline components/flowerFooter/FlowerFooter.tsx:74]: **thought:** We might want to start keeping hardcoded colors and other similar styles in the theme to improve consistency, do you think we should add this color to the `defaultTheme`?

File: components/flowerFooter/FlowerFooter.tsx [hunk 3]
```diff
@@ -1,40 +1,28 @@
...
-  useMediaQuery,
-  useTheme
-} from '@mui/material';
-const catFooterImage = `${process.env.NEXT_PUBLIC_SUPABASE_URL}/storage/v1/object/public/considerate%20cat%20assets/catNamedShoe.png`;
+import { Box, Typography, useMediaQuery, useTheme } from '@mui/material';
 
-interface flowerFooterProps {
-  sx?: SxProps;
-}
+const catFooterImage = `${process.env.NEXT_PUBLIC_SUPABASE_URL}/storage/v1/object/public/considerate%20cat%20assets/catNamedShoe.png`;
```
Comment (arthvadrr) [inline components/flowerFooter/FlowerFooter.tsx:5]: **thought:** Do you think we should optimize this image and the one for the footer background and put it in the project files?

File: components/flowerFooter/flowerFooter.stories.tsx [hunk 4]
```diff
@@ -4,9 +4,22 @@ import FlowerFooter from './FlowerFooter';
...
+      </div>
+    )
+  ]
 } as Meta<typeof FlowerFooter>;
 
-const Template: StoryFn<typeof FlowerFooter> = () => <FlowerFooter sx={{}} />;
+const Template: StoryFn<typeof FlowerFooter> = () => <FlowerFooter />;
 
 export const Default = Template.bind({});
+
```
Comment (arthvadrr) [inline components/flowerFooter/flowerFooter.stories.tsx:19]: **praise:** I like the mobile view in Storybook!

**Review by arthvadrr — CHANGES_REQUESTED** [review]

File: components/flowerFooter/FlowerFooter.tsx (hunk 2 above)
Comment (madeleine-patience) [reply components/flowerFooter/FlowerFooter.tsx:74]: Yes not a bad idea, i wasn't sure how far we usually would go in situations like this as i imagine considerate cat having a ton of colors by the end of it. But good idea to keep consistent. Will add.

File: components/flowerFooter/FlowerFooter.tsx (hunk 3 above)
Comment (madeleine-patience) [reply components/flowerFooter/FlowerFooter.tsx:5]: Yes i think it is a good idea, have we done this yet ? I thought we were pulling all the images directly from supabase.

Comment (arthvadrr) [reply components/flowerFooter/FlowerFooter.tsx:5]: We have, I've been putting optimized component images in the `public` dir.

Comment (madeleine-patience) [reply components/flowerFooter/FlowerFooter.tsx:5]: Got it, changing this.

Also lol, did you name this or did i?
image

Comment (madeleine-patience) [reply components/flowerFooter/FlowerFooter.tsx:5]: Would you put both footer images in public? How do you decide what goes in public and what comes from supabase?

File: components/flowerFooter/FlowerFooter.tsx (hunk 2 above)
Comment (madeleine-patience) [reply components/flowerFooter/FlowerFooter.tsx]: Done!

File: components/flowerFooter/FlowerFooter.tsx (hunk 3 above)
Comment (madeleine-patience) [reply components/flowerFooter/FlowerFooter.tsx]: Done :)

**Review by arthvadrr — APPROVED** [review]

**Pull request merged**